gradle/libs.versions.toml

```diff
@@ -0,0 +1,68 @@
+# which runs on JDK 11.
+layoutlib = "2021.2.1-patch1-fa3aa65"
+layoutlibPrebuiltSha = 'fa3aa65'
+
+# Note: there are some compose/androidx versions left hardcoded in `paparazzi/paparazzi/build.gradle`,
+# because they refer to JAR files names in the `paparazzi/paparazzi/libs` folder.
+
+[libraries]
+androidx-annotations = { module = "androidx.annotation:annotation", version = "1.3.0" }
+androidx-lifecycleCommon = { module = "androidx.lifecycle:lifecycle-common", version = "2.4.0" }
+
+bytebuddy-agent = { module = "net.bytebuddy:byte-buddy-agent", version.ref = "bytebuddy" }
+bytebuddy-core = { module = "net.bytebuddy:byte-buddy", version.ref = "bytebuddy" }
+
+compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "compose-compiler" }
+composeUi-material = { module = "androidx.compose.material:material", version = "1.2.1" }
+
+guava = { module = "com.google.guava:guava", version = "31.0.1-jre" }
+
+jcodec-core = { module = "org.jcodec:jcodec", version.ref = "jcodec" }
+jcodec-javase = { module = "org.jcodec:jcodec-javase", version.ref = "jcodec" }
+
+kotlin-bom = { module = "org.jetbrains.kotlin:kotlin-bom", version.ref = "kotlin" }
+kotlinx-coroutines-core = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-core", version = "1.6.4" }
+
+kxml2 = { module = "kxml2:kxml2", version = "2.3.0" }
+
+layoutlib-native-jdk11 = { module = "app.cash.paparazzi:layoutlib-native-jdk11", version.ref = "layoutlib" }
+layoutlib-native-linux = { module = "app.cash.paparazzi:layoutlib-native-linux", version.ref = "layoutlib" }
+layoutlib-native-macArm = { module = "app.cash.paparazzi:layoutlib-native-macarm", version.ref = "layoutlib" }
+layoutlib-native-macOsX = { module = "app.cash.paparazzi:layoutlib-native-macosx", version.ref = "layoutlib" }
+layoutlib-native-windows = { module = "app.cash.paparazzi:layoutlib-native-win", version.ref = "layoutlib" }
+
+moshi-adapters = { module = "com.squareup.moshi:moshi-adapters", version.ref = "moshi" }
+moshi-core = { module = "com.squareup.moshi:moshi", version.ref = "moshi" }
+moshi-kotlin = { module = "com.squareup.moshi:moshi-kotlin-codegen", version.ref = "moshi" }
+
+okio = { module = "com.squareup.okio:okio", version = "3.1.0" }
+
+tools-common = { module = "com.android.tools:common", version = "27.1.2" }
+tools-layoutlib = { module = "com.android.tools.layoutlib:layoutlib-api", version = "27.2.2" }
+tools-ninepatch = { module = "com.android.tools:ninepatch", version = "30.1.2" }
+tools-sdkCommon = { module = "com.android.tools:sdk-common", version = "26.6.4" }
+
+# Test libraries
+assertj = { module = "org.assertj:assertj-core", version = "3.23.1" }
+junit = { module = "junit:junit", version = "4.13.2" }
+testParameterInjector = { module = "com.google.testparameterinjector:test-parameter-injector", version = "1.8" }
+truth = { module = "com.google.truth:truth", version = "1.1.3" }
+
+# Plugins
+plugin-android = { module = "com.android.tools.build:gradle", version = "7.1.2" }
```

Comment:
extract agp back as a version.  yes, it's only used once, but this is a deliberate exception.

Reply:
conveniently starts with `a` :)